### DISCUSSION TITLE
No apperance of the subtitle bar in the Main window

### DISCUSSION THREAD

**Discussion by FenderRhodes** [kannagi0303/yt-dlp-gui#118] [Q&A]
Hi there,

First of all, I would like to show my gratitude to the developer who provides the yt-dlp-gui as free in public.
I've just got started with the yt-dlp-gui in order to download a streaming video, audio and subtitle.
I've been looking for a GUI based app because I'm not familiar with command based one such as yt-dlp and ffmpeg.

Both The Stream Detector for Chrome Addon and Firefox addon have been installed. It detects multiple HLS URLs when it is logged and accesses content at one of my favorites streaming services such as tubi tv and Roku channel. The URLs are fed to either  JDwonloader2 or the yt-dlp-gui and download the video and audio separately but I never get any subtitles (English) so far..

In this system, an english subtitle(.srt) can be downloaded along with video(.mp4) and audio(.m4a) when it is logged and access a content with YouTube because no need to detect HLS URLs with the The Stream Detector addon. It is just a copy of the URL of Chrome or Firefox and feeds it to JDwonloader2 or the yt-dlp-gui to download all the necessary media files (video, audio and subtitle) without an issue.  

However, I've found no appearance of the subtitle bar in the Main window when it is activated. It must be there. So I need your help to resolve the issue.
  
![ScreenShot_202308081229](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/f7bb04d5-95a2-4e0e-81fe-0cfebacfbcb5)

I've found the same in the Main window when I attempted to feed a target URL in YouTube as follows.

![ScreenShot_20230808104948](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/0c50d70f-036d-4c5a-a512-d6729f3bd3b4)

Here is the screen shot where the yt-dlp-gui is located as follows.

![ScreenShot_20230808105033](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/bb702871-7d9f-4ed6-84ff-aabffc214a48)

You may need to check the property of the yt-dlp-gui as follows.

![ScreenShot_20230808111102](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/ddc1807b-7c9c-4e32-a8d4-60ae16dc8ab8)

You may also need to check the preference of the yt-dlp-gui as follows.

![ScreenShot_202308081243](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/6955fbb4-7d9a-49d0-b380-02332826f36e)

![ScreenShot_20230808105054](https://github.com/kannagi0303/yt-dlp-gui/assets/141695357/109f6d75-20a6-4fd1-891b-907919565dfd)

Thank you for your help in advance.

Warm Regards,

**Comment by kannagi0303** [comment]
I can't view the content of the video from the URL in your screenshot, basically it won't contain "auto-generated" subtitles.

If the video has subtitles, the drop-down menu on the first page will display subtitle items

**Comment by FenderRhodes** [comment]
Hi,

Thank you for your reply.
 
You mean that the reason why it doesn't display the subtitle along with Video and Audio in the drop-down menu on the first page (Main) is because a given URL has a subtitle which is auto generated, otherwise, it has no subtitles file or link included in the URL. It should be displayed the same if the given URL has a subtitle file or link.

Then it will display Video and Audio only without Subtitle in the drop-down menu on the first page or Main window when the yt-dlp-gui is activated. Is it OK for my understanding of your reply?

The target URL in the screenshot was here as follows. It is just a reference.
“Barbara School” Episode #12 - Personal Experiences with Extraterrestrials (ETs)
https://www.youtube.com/watch?v=RJpjnaR8nYE

Thanks again.

**Reply by kannagi0303** [reply]
If the URL contains subtitles (excluding auto-generated subtitles), the Subtitles menu will appear below Video and Audio in the GUI.

As in the URL you provided, the video only has YouTube auto-generated subtitles, so the Subtitles option will not appear.
